master/docs/manual/customization.rst

```diff
@@ -175,6 +175,26 @@ A simple ``prioritizeBuilders`` implementation might look like this::
 
     c['prioritizeBuilders'] = prioritizeBuilders
 
+If the change frequency is higher than the turn-around of the builders,
+the following approach might be helpful:
+
```

Comment:
isn't it trying to workaround the issue that is fixed here: https://github.com/buildbot/buildbot/pull/3685 ?

Reply:
I don't think so. In my case, the master is not busy at all. It's just that the default priority function interacts badly with the collapse logic. New changes will have the same submission time for all builders of that scheduler and a collapsed build will retain its submission time. As such, the later builders are never considered as long as there is a steady flow of new changes.